flopy/modflow/mfdis.py

```diff
@@ -939,11 +939,13 @@ def load(f, model, ext_unit_dict=None, check=True):
             print('   loading delr...')
         delr = Util2d.load(f, model, (ncol,), np.float32, 'delr',
                            ext_unit_dict)
+        delr = delr.array.reshape(ncol)
         # dataset 4 -- delc
         if model.verbose:
             print('   loading delc...')
         delc = Util2d.load(f, model, (nrow,), np.float32, 'delc',
                            ext_unit_dict)
+        delc = delc.array.reshape(nrow)

```

Comment:
Not sure why delc is being converted to a numpy array instead of leaving as a Util2d object.